src/Framework/NativeMethods.cs

```diff
@@ -1148,26 +1152,131 @@ internal static bool MakeSymbolicLink(string newFileName, string exitingFileName
                 flags |= SymbolicLink.AllowUnprivilegedCreate;
             }
 
-            symbolicLinkCreated = CreateSymbolicLink(newFileName, exitingFileName, flags);
+            symbolicLinkCreated = CreateSymbolicLink(newFileName, existingFileName, flags);
             errorMessage = symbolicLinkCreated ? null : Marshal.GetExceptionForHR(Marshal.GetHRForLastWin32Error()).Message;
         }
         else
         {
-            symbolicLinkCreated = symlink(exitingFileName, newFileName) == 0;
+            symbolicLinkCreated = symlink(existingFileName, newFileName) == 0;
             errorMessage = symbolicLinkCreated ? null : "The link() library call failed with the following error code: " + Marshal.GetLastWin32Error();
         }
 
         return symbolicLinkCreated;
     }
 
+    internal static string GetRelativePath(string relativeTo, string path)
```

Comment:
Did you write this method yourself? It looks correct to me, as far as I can tell, but it's substantially longer than an answer using URIs:
https://stackoverflow.com/questions/275689/how-to-get-relative-path-from-absolute-path

Might that work? My primary concern is that it'd be pretty easy to get this slightly wrong in an important way and have it fail. That is, I think it's right, but I'm not confident.

Reply:
As the comment in the method says, it is based upon [.NET 7 Path.cs](https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/System/IO/Path.cs#L843)
The unit test cases also come from .NET7.

I updated the code to using the `Uri.MakeRelativeUri` method as used in the article, but many of the unit test cases failed. Even after adding a trailing slash, still many tests failed. Seeing all the comments on the article, I'm not the only one. Uri also have a far more restricted character set than files. 

I therefore think the original code is appropriate.

Why does msbuild need a build target for net472 and netstandard20? Do you update msbuild for older SDKs?